be/src/runtime/export_sink.cpp

```diff
@@ -142,10 +142,10 @@ Status ExportSink::gen_row_buffer(TupleRow* row, std::stringstream* ss) {
                     (*ss) << reinterpret_cast<PackedInt128*>(item)->value;
                     break;
                 case TYPE_FLOAT:
-                    (*ss) << *static_cast<float*>(item);
+                    (*ss) << std::to_string(*static_cast<float*>(item));
```

Comment:
Could you please add a comment to tell the reader why it was converted to a string first?

For example the following comment：

To prevent loss of precision on `float` and `double` types, they are converted to strings before output.
For example: For a double value `27361919854.929001`, the direct output of using `std::stringstream` is` 2.73619e + 10`, and after conversion to a string, it outputs `27361919854.929001`

Reply:
Please use FloatToBuffer(gutil/strings/numbers.h) to print float and double value